api/lql_execute.go

```diff
@@ -38,18 +38,34 @@ const (
 	QueryEndTimeRange   ExecuteQueryArgumentName = "EndTimeRange"
 )
 
+type ExecuteQueryOptions struct {
+	Limit *int `json:"limit,omitempty"`
+}
+
+func (eqo ExecuteQueryOptions) Validate() error {
+	if eqo.Limit == nil {
+		return nil
+	}
+	if *eqo.Limit >= 1 {
+		return nil
+	}
+	return errors.New("limit must be at least 1")
+}
+
```

Comment:
Do we need to implement all of that? Shouldn't this be enforced by the APIs?

Reply:
Testing passing a negative value does show:
```
[400] Incorrect configuration of field options.limit: must be greater than or equal to 1
```

Reply:
Up to you to leave it, this is just a nitpick since you already implemented the logic/code.

Reply:
I worked this up based on some early go-sdk feedback you had given about reducing API calls if we're able to validate client side.  The pro is the ability to reduce server-side calls, but the con is having to keep in-line with platform validation.

If you're good with it I might chose to remove this in favor of server-side validation...

David